lib/src/wso2/wso2-api/handler/wso2-v1.ts

```diff
@@ -349,7 +360,8 @@ export const openapiSimilarWso2 = (
   if (
     subOpenapi.info.title !== wso2Openapi.info.title ||
     subOpenapi.paths.length !== wso2Openapi.paths.length ||
-    subOpenapi.servers?.length !== wso2Openapi.servers?.length ||
+    // TODO: wso2 fetch seems to not return the servers
```

Comment:
It's not clear for me what we need to do with this "TODO"? Maybe it's only a comment. If not, please create an issue so we can work on it later.